modules/homebrew/README.md

```diff
@@ -23,15 +23,16 @@ Aliases
   - `brewl` lists installed formulae.
   - `brewo` lists brews which have an update available.
   - `brews` searches for a formula.
-  - `brewu` updates and upgrades Homebrew packages and formulae.
```

Comment:
I'm not 100% sure this is a readability improvement but I get what it's trying to do so it's probably fine

Reply:
Agreed! All `brew` operations update Homebrew implicitly anyway. How does the suggested change sound?